src/Manager/UserManager.php

```diff
@@ -64,16 +66,18 @@ public function transferUserToPartner(User $user, Partner $partner): void
             $this->urlGenerator->generate('back_dashboard') :
             $loginLink;
 
-        $this->notificationService->send(
-            NotificationService::TYPE_ACCOUNT_TRANSFER,
-            $user->getEmail(),
-            [
-            'btntext' => User::STATUS_ACTIVE === $user->getStatut() ? 'Accéder à mon compte' : 'Activer mon compte',
-            'link' => $link,
-            'user_status' => $user->getStatut(),
-            'partner_name' => $partner->getNom(),
-        ],
-            $user->getTerritory()
+        $this->notificationMailerRegistry->send(
```

Comment:
affichage de l'erreur suivante lors de la tentative de transférer un utilisateur (et je ne comprend pas trop pourquoi)
`App\Manager\UserManager::transferUserToPartner(): Argument #1 ($user) must be of type App\Entity\User, null given, called in /app/src/Controller/Back/BackPartnerController.php on line 157`

Reply:
toujours le cas

Reply:
C'est corrigé

Reply:
j'ai toujours l'erreur

Reply:
Pas d'erreur pendant le test

Reply:
c'est ok, pas d'erreur, mail reçu